src/js/gallery.js

```diff
@@ -25,13 +26,30 @@ const searchParams = () =>
     page: page,
   });
 
-let wasFirstFetch = false;
+// let wasFirstFetch = false;
 const options = {
-  rootMargin: '0px',
-  threshold: 0.1,
+  // rootMargin: '0px',
+  // threshold: 0.1,
+  rootMargin: '550px',
```

Comment:
Trochę magiczna wartość. Będziemy musieli się później zastanowić jak to mądrzej zapisać ;)